include/vcpkg/base/chrono.h

```diff
@@ -54,12 +53,12 @@ namespace vcpkg
         void to_string(std::string& into) const;
 
     private:
-        std::chrono::high_resolution_clock::time_point m_start_tick;
+        std::atomic<std::chrono::high_resolution_clock::time_point> m_start_tick;
```

Comment:
This change broke the build with clang++-8 on Ubuntu 18.04. `std::atomic<T>` requires `T` to be nothrow constructible, but `std::chrono::high_resolution_clock::time_point` isn't.

Reply:
(More precisely, the `std::atomic<T>` constructor is `noexcept` before C++20, and this has consequence for the member of type `T`.)